### ISSUE TITLE
Look at different ways of fenced blocks

### ISSUE THREAD

**Issue by kizu** [shower/jekyller#21]
Could I convert blocks that are not supported right now to proper code?

Right now the version at Pages don't understand `~~~ sql` type of fenced blocks :(

And kramdown don't know about ``` also.

**Comment by audreyt** [comment]
+1 for this; compare http://www.audreyt.org/newdict/jekyller/_public/ vs http://audreyt.github.io/jekyller/ for a real-world example.

I wonder if it's easiest to allow the ``` syntax, and pre-process it into ~~~ if the engine is kramdown?

BTW, the presentation above was very well received, and thx so much for Jekyller -- it cured my Keynote.app habit since 2006. :-)

**Comment by kizu** [comment]
Yep, I think I'll try to render the kramdown right from the jekyll, however I'm now sure if the pygments would work in this case.

**Comment by kizu** [comment]
Also, good to see Jekyller to be used for an actual presentation! :)

**Comment by audreyt** [comment]
I'm very happy with it so far. :-)

I think it's fine to make pygments optional, as GitHub Pages may not have CodeRay installed (very happy to be corrected if it does). A locally-rendered Jekyller can receive syntax highlighting, while a github.io one at least gets the formatting correct.
